Views/General/Feed/Card.swift

```diff
@@ -89,19 +89,21 @@ struct Card: View {
             VStack(alignment: .trailing) {
                 ForEach(trailingPropertyList, id: \.self) { info in
                     Text(info)
-                        .foregroundColor(.white)
+                        .foregroundColor(.accentColor)
+                        .fontWeight(.semibold)
                         .padding(4)
                         .background(
                             RoundedRectangle(cornerRadius: 8)
+                                .fill(.white)
```

Comment:
Are you satisfied with how this view appears in both dark & white mode?

I would say in light mode that looks like a "hole" and too bright for dark mode, just making sure you're actually wanting this.